packages/constellation/src/components/Compositions/Table/Cells/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './BaseCell'
+export * from './ButtonCell'
+export * from './CheckboxCell'
+export * from './IconCell'
+export * from './InformationalCell'
+export * from './TextCell'
```

Comment:
I think these pre-designed cells are a nice way of prepackaging the cell designs, but ultimately I think the consumer should be able to compose their own cells using only `BaseCell`. From what I can tell this is the case, so I think this is looking good so far 👍

Reply:
I was assuming that we would not be allowing custom cells however down the line it would probably be very helpful to have that option.